mart/attack/adversary_in_art.py

```diff
@@ -5,8 +5,12 @@
 # agreement between Intel Corporation and you.
 #
 
+from typing import Any, List, Optional
+
 import hydra
+import numpy
 import torch
+from art.estimators.object_detection.object_detector import ObjectDetector
```

Comment:
Do you think it's worth guarding against this import using `typing.TYPE_CHECKING`? It seems better to guard imports that are only used when typing checking.

Reply:
I retract my comment :)